src/main/kotlin/io/github/mojira/arisa/modules/commands/CommandDispatcher.kt

```diff
@@ -135,6 +136,14 @@ fun getCommandDispatcher(
                         }
                 )
 
+        val makePrivateCommandNode =
+            literal<CommandSource>("${prefix}_MAKE_PRIVATE")
```

Comment:
This doesn't check that the user is a helper?

Reply:
The CommandModule only sends helper or staff comments to the dispatcher